spec/controllers/v2/tailorings_controller_spec.rb

```diff
@@ -253,6 +247,42 @@
       end
     end
 
+    context 'with authorization only via cert_auth' do
+      before do
+        allow_any_instance_of(V2::ApplicationController).to receive(:rbac_allowed?).and_call_original
+        allow_any_instance_of(Insights::Api::Common::HostInventory).to receive(:hosts).and_return('results' => [:foo])
```

Comment:
In the first case you definitely not need the `_any_instance_of`, look at the mocking example from above. For the second one, you can extract the host inventory API lookup into a separate method and mock that. Calling a global mock is both dangerous and slow.